server/src/test/java/io/crate/integrationtests/ViewsITest.java

```diff
@@ -215,4 +219,41 @@ public void test_creating_a_self_referencing_view_is_not_allowed() {
             )
         );
     }
+
+    @Test
+    @UseRandomizedSchema(random = false)
+    @UseHashJoins(1)
+    public void debug_test_to_be_removed() {
```

Comment:
I think we ought to have a test case that covers the bug/fix. Is the plan to re-add this as a planner test?

Reply:
Yep, 
> I will replace an Itest by a Planner test once approach is finalized.